main/dconf/APKBUILD

```diff
@@ -1,32 +1,28 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dconf
-pkgver=0.26.0
+pkgver=0.32.0
 pkgrel=1
 pkgdesc="A low-level configuration system."
 url="http://live.gnome.org/dconf"
```

Comment:
```suggestion
url="https://wiki.gnome.org/action/show/Projects/dconf"
```

Reply:
It's https://wiki.gnome.org/Projects/dconf